src/Middleware/HttpLogging/src/HttpLoggingExtensions.cs

```diff
@@ -0,0 +1,15 @@
+// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+
+using System;
+using Microsoft.Extensions.Logging;
+
+namespace Microsoft.AspNetCore.HttpLogging
+{
+    internal static class HttpLoggingExtensions
+    {
+        static HttpLoggingExtensions()
```

Comment:
TODO, adding logging at debug/trace levels (forgot about this 😄 )

Reply:
lol